src/test/java/pages/SongsPage.java

```diff
@@ -0,0 +1,63 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class SongsPage extends BasePage {
+
+    By allSongTabLocator = By.cssSelector("a.songs");
+    By selectFirstSonglocator = By.xpath("//*[@id='songsWrapper']//div[1]/table/tr[1]/td[2]");
+    By soundBar = By.xpath("//div[@data-testid='sound-bar-play']");
+    By selectFirstSongField = By.xpath("//*[@id='songsWrapper']//div[1]/table/tr[1]/td[2]");
+    By addSelectedSong = By.cssSelector("button.btn-add-to");
+    By selectCreatedPlaylist = By.xpath("//*[@id='songsWrapper']/header/div[3]/div/section[1]/ul/li[5]");
+    By songAddedMessageLocator = By.cssSelector("div.alertify-logs.top.right");
```

Comment:
I recommend:  **make all the locators and WebElements private**, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");